scripts/core/superdesk-api.d.ts

```diff
@@ -3657,6 +3659,8 @@ declare module 'superdesk-api' {
          * (it will be rendered in different DOM locations depending if field is in header or content section)
          */
         miniToolbar?: JSX.Element;
+        sectionClassNames?: IAuthoringSectionClassNames;
+        fieldId?: IAuthoringField['id'];
```

Comment:
You shouldn't be adding properties to interfaces that are only used for testing. In this case, field ID is available in `authoring-section-field.tsx` - and is being used to add a data-test-id. This change should not be needed.